### PR TITLE
Clean up recon copy

### PR DESCRIPTION
**Pull request by oed**
Cleans up copy outside of the spec section.

- [x] Summary
- [x] Abstract
- [x] Motivation 
- [x] Rationale
- [x] Backwards compat.
- [x] Implementation
- [x] Security cons.
- [x] Appendix

In addition I also renamed *separator-key/value* to *sort-key/value*.

### CODE REVIEW

File: CIPs/cip-124.md [hunk 1]
```diff
@@ -6,69 +6,40 @@ discussions-to: https://forum.ceramic.network/t/cip-124-recon-tip-synchronizatio
...
-edited: 2023-05-24
+edited: 2023-05-25
 ---
 ## Simple Summary
 
 <!--Provide a simplified and layman-accessible explanation of the CIP.-->
-Recon is a sets reconciliation protocol to be used for synchronization of stream tips in ceramic network.
-Stream Sets bundle a number of streams together so the ceramic node with a common interest in those streams can synchronize efficiently.
-Once a set of streams is bundled into a stream set Recon can be used to sync ranges within those sets.
+Recon is a sets reconciliation protocol to be used for synchronization of stream events in ceramic network.
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-124.md:14]: yep, typo.

File: CIPs/cip-124.md [hunk 2]
```diff
@@ -6,69 +6,40 @@ discussions-to: https://forum.ceramic.network/t/cip-124-recon-tip-synchronizatio
...
 
-Ceramic nodes also need to have a way to find the other nodes interested in the stream set so that they can synchronize with them.
-Recon will rely on gossiping about nodes existence and interests. When two nodes connect they can share lists of known nodes and their interests.
-
-By gossiping about ranges of streams ceramic nodes that are arbitrarily out of sync can synchronize all events of mutual interest. Nodes that are in sync or nearly in sync can send each other very little bandwidth. Nodes can avoid sending stream event announcements to nodes that have no interest in the stream ranges.
+Finally, nodes also need a way to find other nodes interested in the stream set or sub-range, so that they can synchronize with them. Recon relies on nodes gossiping their interest to peers, as well as keeping a list of their peers interest. This way nodes that are in sync, or nearly in sync, stay in sync with very little bandwidth. Nodes can also avoid sending stream event announcements to nodes that have no interest in the stream ranges.
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-124.md:29]: I did not know this. Thanks!

File: CIPs/cip-124.md [hunk 3]
```diff
@@ -6,69 +6,40 @@ discussions-to: https://forum.ceramic.network/t/cip-124-recon-tip-synchronizatio
...
 
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature.-->
 
 ### Stream Set
 
-A stream set is defined by a value called `separator-key` which determines the sort ordering. A stream is considered to be part of a stream set if its InitEvent contains a header field that maps to the `separator-key`. The value of this field is referred to as the `separator-value`. For example, a stream set containing all *Models* and *Model Instance Documents* is defined with the *separator-key* `'model'`.
+A stream set is defined by a value called `sort-key` which determines the top level sort ordering. A stream is considered to be part of a stream set if its InitEvent contains a header field that is equal to the `sort-key`. The value of this field is referred to as the `sort-value`. For example, a stream set containing all *Models* and *Model Instance Documents* is defined with the *sort-key* `'model'`.
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-124.md:36]: The change is to make it easier to understand yes. I think we need to discuss if we actually want to use `sep` as a special field or not, but IMO that is out of scope of this PR.

File: CIPs/cip-124.md [hunk 4]
```diff
@@ -103,7 +74,7 @@ Where:
 
 * `networkId` is a number as defined in the [networkIds table](../tables/networkIds.csv)
 
-* `separator-value` is based on a user provided value for *separator-key* and *separator-value*
+* `sort-value` is based on a user provided value for *sort-key* and *sort-value*
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-124.md:77]: As far as I understand, this CIP doesn't really go into if you sort events as they come in or not. I could also imagine a world in which an event is only added to an index if it's received in that particular range.

File: CIPs/cip-124.md [hunk 5]
```diff
@@ -291,154 +262,85 @@ eventId = concatBytes(
 ## Rationale
 <!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
 
-### Why the properties in eventIds
+### Interest ranges
 
-* Why specific lengths?
-* Size overhead?
+At the core of the design of recon there is the concept of an interest range. Essentially a range is an ordered list of events across multiple streams. By using ranges to specify interest we allow nodes to have partial overlap with each other. This is useful because it removes the need for the network to have a uniform topology. Instead each node can chose to subscribe to as many or as few streams as they like to.
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-124.md:267]: Ok, will update this!

**Review by AaronGoldman — APPROVED** [review]

**Pull request merged**